src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -2,8 +2,56 @@ export const INSTALLED = 'Installed' as const;
 export const NOT_INSTALLED = 'Not Installed' as const;
 export const PENDING = 'Pending' as const;
 
-export const colors = {
+export const COLORS = {
   [INSTALLED]: 'success',
   [NOT_INSTALLED]: 'gray2',
   [PENDING]: 'yellowOrange',
-};
+} as const;
+
+/**
+ * Integrations in the integration directory should be sorted by their popularity (weight).
+ * These weights should be hardcoded in the application itself.
+ * We can store this in a map where the key is the integration slug and the value is an integer and represents the weight.
+ * The weights should reflect the relative popularity of each integration.
+ */
```

Comment:
tiny nit: I tend to wrap my comments to 80 chars. Generally If I am splitting lines I'll make a new paragraph with a blank line between

Reply:
@EvanPurkhiser I'm splitting lines just so that I don't have to scroll horizontally when reading comments. Is that ok?

Reply:
You can use `option-z` to toggle word-wrap on VS Code. It might be a good idea to add rulers for 80/100/120 chars too.

Reply:
@leedongwei cool thanks! did not know about this!

Reply:
Ehh @NisanthanNanthakumar 

@EvanPurkhiser saying that you should manually add a line-break if a LoC is more than 80 chars.

My comment is a suggestion that if you want to avoid scrolling horizontally, you can use word-wrap on VS Code. If the line exceeds 80 chars, the rulers will help you see that. The VS Code word-wrap is purely cosmetic.

Reply:
I like to have my comments wrapped at 80 fwiw so that I can have lots of panes open to around 80 chars in width.

Reply:
https://marketplace.visualstudio.com/items?itemName=stkb.rewrap maybe this is usefl

Reply:
That plugin is lit.